HideAndSeek.js

```diff
@@ -4,11 +4,11 @@ function publishReview() {
     var button = reviewMenu.querySelector('button');
 
     button.onclick = function() {
-      var author = document.querySelector('.author').innerText;
+      var author = document.querySelector('.header-nav-current-user').querySelector('strong').innerText;
       var files = loadData()[getPullRequestId()]['files'];
       //check button and then check if they want to completes
       var meta = { author: files };
-      var rawMeta = metaData(localStorageKey, meta)
+      var rawMeta = metaData(localStorageKey, meta);
```

Comment:
jhgfjhg

Reply:
i think this is garbage

Reply:
asdf

Reply:
asdfddfdf

Reply:
kdkd

Reply:
garbage

Reply:
asdf

Reply:
<img src="https://emoji.slack-edge.com/T039B0HP2/party_dumpster_fire/b3fcce41cd8504d8.gif" alt="party_dumpster_fire" title="party_dumpster_fire" width="25">

Reply:
test